test/deploy.smart.wallet.test.ts

```diff
@@ -0,0 +1,70 @@
+import { RelayingServices, SmartWallet } from '../src';
```

Comment:
please see @antomor's last comment, in it he's provided links for old suggestions on how to word these tests in a more intelligible way